### ISSUE TITLE
Mouse sensitivity fluctuates when G-Helper is opened

### ISSUE THREAD

**Issue by GastonKhouri** [seerge/g-helper#3166]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

When G-Helper is opened, it causes mouse sensitivity issues in Call of Duty, where the mouse movement randomly slows down during gameplay. The sensitivity becomes unpredictable, making it difficult to play. This issue happens consistently whenever G-Helper is running, and I have tested with multiple mice to confirm that it's not a hardware issue. The problem persists even after restarting the laptop, but it goes away if I completely shut down the laptop and turn it back on.

### How to reproduce the bug?

1. Launch Call of Duty
2. Open G-Helper
3. Play the game for a few minutes
4. Notice the random mouse sensitivity drops

### Logs

[log.txt](https://github.com/user-attachments/files/17121565/log.txt)

### Device and Model

ROG Strix G16 2023 (G614JI)

### Additional information.

This issue only occurs when G-Helper is running. The issue persists after closing the app and restarting the laptop but goes away when the laptop is fully shut down and turned back on.

### Armoury Crate

Uninstalled

### Asus Services

None

### Version

0.191.0

### OS

Windows 11 Home

**Comment by seerge** [comment]
@GastonKhouri hello,

G-Helper [is not a driver](https://github.com/seerge/g-helper?tab=readme-ov-file#-important-notice), it doesn't interact with your hardware in realtime anyhow :) 

As I understand from the log, your mouse is not even from Asus ? If so - G-Helper is not even able to set any settings for it. So I extremely doubt it's related. You can simply quit app - to verify if your mouse behavior changes after that or not.

Do you have any other apps running that can be related to controller / mouse / etc ?

**Comment by seerge** [comment]
P.S. Did you try plugging mouse into different USB-port ? (i see that windows notifies app about constant changes in USB devices, it looks more like your mouse is simply disconnecting periodically)

**Comment by GastonKhouri** [comment]
Hello @seerge,

Thanks for your quick response and suggestions!

I’ve done some more testing and found that the issue only occurs when I have Turbo mode enabled. When using Balanced mode, the problem disappears. This makes me think it could be related to the power supply to the mouse while in Turbo mode.

I’ve tested this with two different Razer mice (Basilisk and Viper V3 Pro). Initially, I had the mouse connected through a hub, but after connecting it directly to the laptop, the issue persisted in Turbo mode. However, when in Balanced mode, the issue doesn’t happen at all. The only difference between my profiles is the fan settings, which are the default ones. Both the Windows power mode and CPU boost settings are the same in both profiles (Best performance and Aggressive, respectively).

Thanks a lot for the quick responses! For now, I'll play using Balanced mode and assume it's a power issue when running the fans at full speed. Hopefully, this helps anyone in the future with a similar problem. They should try switching to Balanced mode to see if it helps.

Cheers!

**Comment by seerge** [comment]
@GastonKhouri well, it could be some strange hardware thing, like when device uses more power, motherboard is not able to handle USB properly or something like that. I obviously can't influence that as this is not cause of G-Helper in a first place, but since in Balanced it works fine - you can try to tinker more with the settings in Turbo (for example set CPU power limits?) to narrow down the culprit.

Was your USB-hub connected to the same port as you connect mouse directly ?

**Comment by GastonKhouri** [comment]
Hello again,

First, let me share what I currently have connected to my laptop:

- One of the USB-C ports has my Apple Watch charger
- Another USB-C port is connected to a monitor via DisplayPort
- The HDMI port is connected to a second monitor
- One USB-A port has my keyboard (Wooting 60HE)
- The other USB-A port has a hub where I’ve connected my Stream Deck 15 buttons, my mouse, and my microphone (HyperX QuadCast)

Following your suggestion, I checked the apply power limits option in G-Helper (both set to the default 80W for PL1 and PL2), and the problem disappeared! With that setting applied, the mouse sensitivity issue is gone in turbo mode.

Additionally, I noticed that during those sensitivity drops, the FPS in Call of Duty would drop from 165 to around 130, so it seems likely that this is indeed related to the power supply provided by the motherboard.

Thanks again for your help!
